src/main/java/com/openlattice/data/integration/BulkDataCreation2.java

```diff
@@ -0,0 +1,177 @@
+package com.openlattice.data.integration;
+
+import com.dataloom.mappers.ObjectMappers;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.openlattice.client.serialization.SerializationConstants;
+import com.openlattice.data.EntityKey;
+import com.openlattice.data.serializers.FullQualifiedNameJacksonSerializer;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.io.Serializable;
+import java.util.*;
+
+@SuppressFBWarnings( value = "SECOBDES", justification = "Java serialization for this class only occurs client side." )
+public class BulkDataCreation2 implements Serializable {
+    static {
+        FullQualifiedNameJacksonSerializer.registerWithMapper( ObjectMappers.getJsonMapper() );
+    }
+
+    private Set<Entity>      entities;
+    private Set<Association> associations;
+    private Map<UUID, Set<String>> entitySetIdToEntityIds;
+    private Map<UUID, String> propertyTypeIdToStorageDest;
+
+    @JsonCreator
+    public BulkDataCreation2(
```

Comment:
Don't understand reasoning behind duplication here.

Reply:
Initially I planned on changing BulkDataCreation a lot and wanted to be able to switch between the two for testing, but then ended up just adding two maps to it. So I will eventually just keep the original and add the two maps to it.